int32 robotID
int64 transitionID
bool conditionHolds
bool ack